metadata/africa.json

```diff
@@ -8,7 +8,7 @@
 	"locations": [
 		"Africa"
 	],
-	"whoisServer": "africa-whois.registry.net.za",
+	"whoisServer": "whois.nic.africa",
```

Comment:
This can thrash. Can you remove this and all changes outside of the new zones you’ve added?

The `zonedb` tool lets you filter operations with the `-x` flag which accepts a regular expression to filter by specific zones.

Reply:
these json's came from make update, ok, will remove them

Reply:
You can use the `-x` flag to filter the underlying command.